modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -72,17 +72,25 @@
     '/etc/lightdm/session-cleanup':
       mode    => '0755',
       source  => 'puppet:///modules/ocf_desktop/xsession/lightdm/session-cleanup';
-    '/opt/share/xsession/lightdm-gtk-greeter.po':
-      source  => 'puppet:///modules/ocf_desktop/xsession/lightdm/lightdm-gtk-greeter.po';
   }
 
   # overwrite greeter strings with OCF ones
   package {'gettext':;}
 
+  $po = $::staff_only ? {
```

Comment:
at some point, we should either rename this to `staffOnly` or rename `skipNetworking` and friends to snake case
(but not related to your PR. i'll ticket)